src/pybind/mgr/dashboard/frontend/src/app/core/dashboard-settings/setting-form/setting-form.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="col-sm-12 col-lg-6">
+  <h1 *ngIf="loading"
+      class="jumbotron">
+    <i [ngClass]="[icons.spinner, icons.pulse, icons.large]"
+       class="text-primary"></i>
+    <ng-container i18n>Loading...</ng-container>
+  </h1>
+  <form *ngIf="!loading"
+        name="settingForm"
+        #formDir="ngForm"
+        [formGroup]="settingForm"
+        novalidate>
+    <div class="card">
+      <div i18n="form title|Example: Edit Grafana Setting@@formTitle"
+           class="card-header">{{ actionForm | titlecase }} setting</div>
```

Comment:
I think you need to use a variable for the resource name, otherwise the translation with id `formTitle` will not work correctly.
```suggestion
           class="card-header">{{ actionForm | titlecase }} {{ resource | upperFirst }}</div>
```